### DISCUSSION TITLE
Ticker List

### DISCUSSION THREAD

**Discussion by davide-legolas** [JECSand/yahoofinancials#139] [Q&A]
Hi!
Does exist a function that gives back a full ticker list of the stocks present in yahoo finance ? (basically what we can do with the online yahoo screener?).

I would like to build a screener , starting from the whole ticker list, and then dividing the stocks for industry and sector (i.e. Oil & Gas E&P, 
Energy)  iterating like this:
```
for ticker in ticker_list:
    stock_info = YahooFinancials(ticker)
    profile = stock_info.get_stock_profile_data()
    industry = profile[ticker]['industry']
    sector = profile[ticker]['sector']
```

Problem is, i dont know how to extract a full ticker list given one (or more) specific countries (i.e. united states, united kingdom, australia). Is that possible?
Thank you very much,
Davide

**Comment by lullu57** [comment]
I Have a built a ticker list for Japanese screeners if you would like. But i got it from repeated requests since the tickers for Japan stocks are all numerical based.

**Comment by azataiot** [comment]
I had the same problem, but currently there is no such function or list that you can loop through. For that reason, I have created a [Github repository](https://github.com/azataiot/FE-Data)  that has small portion of what you are ask for. I did not checked or got all the list, but i tried to get TOP 100 for each and every market. (Notice that there are two different branch on that repo, and those two different branch uses two different approach, you might check the branch `next`).

There are some of the works still being done, and if you like we can work on that together, as I need the screener also for my University Project.
